bats/fb-verify-packages.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:ts=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "verify no old SCL packages" {
+  run tSetOSVersion
+  run FOREMAN_VERSION=$(tForemanVersion)
+
+  if tIsRedHatCompatible || tIsCentOSCompatible; then
```

Comment:
CentOS is considerd RH compatible since `/etc/redhat-release` exists there as well.